packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -346,6 +379,41 @@ class RenderParagraph extends RenderBox
     return _textPainter.computeDistanceToActualBaseline(TextBaseline.alphabetic);
   }
 
+  TextSelection? _textSelection;
+
+  @override
+  bool update(Offset start, Offset end) {
+    final Offset localStart = globalToLocal(start);
+    final Offset localEnd = globalToLocal(end);
+    final TextPosition startText = getPositionForOffset(localStart);
```

Comment:
This doesn't quite do what I want if start/end don't really intersect with the text bounding box